lua/lambdaplayers/lambda/weapons/l4d2_golfclub.lua

```diff
@@ -0,0 +1,50 @@
+local random = math.random
+local CurTime = CurTime
+local Rand = math.Rand
+
+table.Merge( _LAMBDAPLAYERSWEAPONS, {
+
+    meathook = {
```

Comment:
This should be renamed

Reply:
Renamed

Reply:
Sorry if this is PR is messy. I never done a Pull request before.